src/utils/src/media.rs

```diff
@@ -0,0 +1,60 @@
+pub struct Box;
```

Comment:
are we using this file still, or did we switch to just using JS? If just JS we can delete this file